tests/deps.node.ts

```diff
@@ -0,0 +1,7 @@
+export function evalCommand() {
+  return ["node", "--eval"];
+}
+
+export function fluentImport() {
+  return 'const { Fluent, defaultWarningHandler } = require("./src/mod.js");';
```

Comment:
This probably also involves DNT magic and I am mainly curious:
Isnt TypeScript using modules and `import` and resolving them to require when CommonJS is used?
Wouldnt it be better to use import here too? Or is there something else I am overlooking / dont know?

Reply:
This is for the tests that dnt runs after transforming the project to JS. So, we cannot use `import` syntax here, the tests will fail.